cl/opinion_page/templates/includes/opinion_tabs_content.html

```diff
@@ -0,0 +1,398 @@
+                    {% endwith %}
+                {% endfor %}
+            </ul>
+        </div>
+    </div>
+{% elif tab == "cited-by" %}
+    {# Cited By #}
+    <div class="tab-pane fade in active" id="cited-by">
+        <h2 class="opinion-section-title jump-link">
+            <span>
+                Cited By ({{ citing_cluster_count|intcomma }})
+                <a href="/feed/search/?type=o&q=cites%3A({{ cluster.sub_opinions.all|OR_join }})" rel="nofollow">
+                    <i class="gray fa fa-rss" title="Subscribe to a feed of citations to this case."></i>
+                </a>
+            </span>
+        </h2>
+        <hr class="hr-opinion">
+
+        {% if citing_cluster_count > 0 %}
+            {% for citing_cluster in citing_clusters %}
+                <article>
+                    <h3 class="bottom serif">
+                        <a href="{{ citing_cluster.absolute_url }}">
+                            {{ citing_cluster.caseName|safe|truncatewords:12|v_wrapper }}
+                            ({{ citing_cluster.dateFiled|date:"Y" }})
+                        </a>
+                    </h3>
+                    <div class="bottom">
+                        <div class="inline-block">
+                            <span class="meta-data-header">Date Filed:</span>
+                            <time class="meta-data-value">
+                                {{ citing_cluster.dateFiled|date:"F jS, Y" }}
+                            </time>
+                        </div>
+                        <div class="inline-block">
+                            <span class="meta-data-header">Status:</span>
+                            <span class="meta-data-value">{{ citing_cluster.status }}</span>
+                        </div>
+                        <div class="inline-block">
+                            <span class="meta-data-header">Citations:</span>
+                            <span class="meta-data-value">{{ citing_cluster.citation|join:", " }}</span>
+                        </div>
+                        <div class="inline-block">
+                            <span class="meta-data-header">Docket Number:</span>
+                            <span class="meta-data-value select-all">{{ citing_cluster.docketNumber }}</span>
+                        </div>
+                    </div>
+                </article>
+            {% endfor %}
+        {% else %}
+            <h3>This case has not yet been cited in our system.</h3>
+        {% endif %}
+
+        <br>
+        <p>
+          <a href="/?q=cites%3A({{ cluster.sub_opinions.all|OR_join }})"
+             rel="nofollow"
+             class="btn btn-default"
+          >View Citing Opinions</a>
+        </p>
+    </div>
+
+{% elif tab == "related-cases" %}
+    {# Related Cases #}
+    <div class="tab-pane fade in active" id="related">
+        <h2 class="opinion-section-title jump-link">Related Cases ({{ related_clusters|length|intcomma }})</h2>
```

Comment:
I noticed the count in the header doesn't match the counts on the tabs. It's a bit confusing. (Screenshot attached). is it possible to make them the same, or just take the count out of the header?

<img width="1248" alt="Screenshot 2025-03-18 at 11 56 58 AM" src="https://github.com/user-attachments/assets/f29c853c-844e-4559-9ac4-abb386dcde33" />

Reply:
Yes, no problem with that, I forgot to clear the ES cache when I took the screenshots.

I delete the cache with these sql statements for cited by and related cases:

```
DELETE FROM django_cache WHERE cache_key LIKE ':1:clusters-cited-es:%';
DELETE FROM django_cache WHERE cache_key LIKE ':1:cited-by-count-es:%';
```

Reply:
This is likely not a cache issue. The related cases are retrieved using the `es_get_related_clusters_with_cache` helper, and the number of results is limited by the `RELATED_COUNT` setting. Consequently, the header count will consistently be 5 unless that setting is modified, as the header reflects the number of results returned from ES. 

https://github.com/freelawproject/courtlistener/blob/8bc5b2f324d3723bdcd49c297302a3a9d7c7ed25/cl/opinion_page/utils.py#L308-L310

Reply:
Thanks eduardo, I took a look again,  I created a lot of dummy data, that is why the ~300 is correct (I cleared the cache and it is slightly bigger).

I noticed that in the html I'm counting the results instead of using the count and as you mentioned it will always be limited to RELATED_COUNT setting, I'll change that so the tab number matches the count in the tab content.

Reply:
It might actually be better not to show a count inside the tab since we're going to do estimated counts in the tab itself. If the estimate has a 5% error rate, it might say 19k, when then the actual number might be 18,412 and that might confuse people. The actual number isn't really that interesting and we've got it in the tab. Skip it in the tab content?

Reply:
Hey Kevin,  thanks for the dummy data script. I got everything populated, but I'm seeing some of the headers in the tabs have duplicate counts. Here's a few examples:

<img width="878" alt="Screenshot 2025-03-18 at 4 46 03 PM" src="https://github.com/user-attachments/assets/cf4caa20-24ea-4fc7-b40a-e257b133c48b" />


<img width="878" alt="Screenshot 2025-03-18 at 4 49 03 PM" src="https://github.com/user-attachments/assets/de5322fa-8370-46b4-aabe-8401558e2725" />

Removing the count from the content makes it much cleaner. Here's how it looks in production

![image](https://github.com/user-attachments/assets/e229c6b9-446c-4202-8daa-995cc8b87c1c)

Reply:
> For all tabs content?

Yes, I think that'll be best, and we can rephrase some of the headers to just remove their counts.

Reply:
Thanks for all your feedback

I fixed the responsive design



https://github.com/user-attachments/assets/4d473561-2d84-4307-8770-9e784183a282




Removed the counts and the extra text from the tab content:

![image](https://github.com/user-attachments/assets/4f875e7a-5d94-4b7c-8ab0-06dfdef95f8e)

![image](https://github.com/user-attachments/assets/2f8150c9-6d63-487f-83aa-86355b1f9456)

![image](https://github.com/user-attachments/assets/dcabb01b-70f2-4d93-916f-3f4de4134d88)

![image](https://github.com/user-attachments/assets/62dd9721-df16-4224-a618-365be26b3924)

Reply:
Looks good to me. Let's see it live, and we can see if we need more explainer, but this should help with performance and accuracy in the meantime. Thanks Kevin!